src/ops/ops.go

```
@@ -750,15 +759,60 @@ func (o *ops) GetNumberOfReboots(ctx context.Context, nodeName, kubeconfigPath s
+		args = append(args, "--image="+img)
+	}
+
+	// Change to workDir before executing command
+	origDir, err := os.Getwd()
+	if err != nil {
+		return "", errors.Wrap(err, "failed to get current directory")
+	}
+	if err := os.Chdir(workDir); err != nil {
+		return "", errors.Wrapf(err, "failed to change to directory %s", workDir)
 	}
+	defer os.Chdir(origDir)
```

golangci-lint (2.12.2)

[error] Error return value of `os.Chdir` is not checked

(errcheck)